apps/web/components/admin/PostEditor.tsx

```
@@ -0,0 +1,233 @@
+              type="button"
+              onClick={() => setShowPreview(!showPreview)}
+              className="text-sm text-primary transition-colors hover:text-primary/90"
+            >
+              {showPreview ? 'Edit' : 'Preview'}
+            </button>
+          </div>
+
+          {showPreview ? (
+            <div
+              className="min-h-[400px] rounded-md border border-border bg-background p-4 prose prose-invert prose-sm max-w-none"
+              dangerouslySetInnerHTML={{ __html: bodyMd.replace(/\n/g, '<br>') }}
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)